src/memtable.c

```diff
@@ -307,15 +307,19 @@ memtable_context_create(platform_heap_id hid,
                         process_fn       process,
                         void            *process_ctxt)
 {
+   platform_memfrag  memfrag_ctxt = {0};
```

Comment:
Here is the 1st example of the new memory allocation interface(s), some of which require an on-stack `platform_memfrag` structure to be declared, named as an extension of the `ptr-handle-variable-name` for which allocation is being done.

Here, we are allocating memory for `ctxt` variable, so the macro expects that a `memfrag_ctxt` be declared on-stack. Otherwise, compilation will fail.

You will see many such patterns of this coding style.

I chose this approach so that we do not perturb the call to memory allocation on L312 below, and still bring-in the requirement of `platform_memfrag *` to memory allocation interfaces.

The approach chosen reduces code churn to some extend in this review packet.